edgexfoundry/edgex-go
# feat(metadata): Implement PATCH /deviceprofile/deviceCommand API

Signed-off-by: Ginny Guan <ginny@iotechsys.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have opened a PR for the related docs change (if not, why?) The Metadata Swagger file is up to update
  <link to docs PR>

## Testing Instructions

1. build core-metadata from this branch and run with other core services
2. send PATCH /deviceprofile/deviceCommand and verify the responses
## New Dependency Instructions (If applicable)
